include/fg/window.h

```diff
@@ -85,6 +85,14 @@ class Window {
         FGAPI void setPos(int pX, int pY);
 
         /**
+           Set the size of the window programmatically
+
+           \param[in] pWidth target width
+           \param[in] pHeight target height
+         */
+        FGAPI void setSize(unsigned pWidth, unsigned pHeight);
```

Comment:
ya, crossed my mind, but since all other methods that are changing internal state are using set* format, i wrote it like this. Also, libraries like glfw, SDL and using setsize only.